fishy/engine/semifisher/fishing_event.py

```diff
@@ -29,11 +29,11 @@ class FishEvent:
     FishingStarted = False
     jitter = False
     previousState = State.IDLE
+    collect_key = 'r'
```

Comment:
Good find, but can you maybe move this to another commit? (`git reset HEAD~` and `git add --patch <file>`)

Reply:
sqash the fixup directly